cli/cmd/configure_monitoring.go

```diff
@@ -52,7 +52,7 @@ keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAM
 		return nil
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
-		if args[0] == "prometheus" {
+		if args[0] == "prometheus" || args[0] == "datadog" {
```

Comment:
:memo: Checking `--service` for `datadog` might not be required because datadog-service at this point does not really use the Service input.